src/IrcCommands/Commands/Join.cpp

```diff
@@ -18,18 +18,20 @@
  * e.g. +k +i +l: has_key, invite_only, has_limit respectively
  * 
  * TODO:
- * (1) +k has_a_key check,
- * (2) +i invite_only check is,
- * (3) +l has_limit check is gonna be implemented
- * (3) implement respective ERR
- * (4) implement resprective RPL
+ * (1) implement a successful joining message
+ * (2) implement resprective RPL messages
+ * (3) multi-channel entry at single command call
+ * Parameters: <channel>{,<channel>} [<key>{,<key>}]
+ * e.g. JOIN #chan,#42,#horizon 
+ * (4) JOIN "0" to all channels at once
```

Comment:
this should mean
```suggestion
 * (4) JOIN "0" to remove client from all channels at once
```
true? You have already implemented the PART-command, it just has to become connected to it.